js-arrays/arrays.js

```diff
@@ -0,0 +1,185 @@
+function isArrayOne(a) {
+  return a instanceof Array;
+}
+
+function isArrayTwo(a) {
+  return Array.isArray(a);
+}
+
+function isArrayThree(a) {
+  return Object.prototype.toString.call(a) === "[object Array]";
+}
+
+function range(a, b, c) {
+  var arr = [];
+  if (b === undefined) {
+    for (var i = 0; i < a; i++) {
+      arr.push(i);
+    }
+  } else if (c === undefined) {
+    for (var i = a; i < b; i++) {
+      arr.push(i);
+    }
+  } else if (c !== 0) {
+    for (var i = a; i < b; i += c) {
+      arr.push(i);
+    }
+  }
+  return arr;
+}
+
+function compactLoop(a) {
```

Comment:
hm, you didn't understand task
compact([1, 0, false, true, {}]) should return [1, true, {}]. it excludes all falsy values